launcher/sdw_updater_gui/Updater.py

```diff
@@ -181,9 +181,16 @@ def _apply_updates_vm(vm):
     will require a reboot after the upgrade.
     """
     sdlog.info("Updating {}".format(vm))
+
+    # We run custom Salt logic for Debian-based TemplateVMs
+    if vm.startswith("fedora"):
+        salt_state = "update.qubes-vm"
+    else:
+        salt_state = "fpf-apt-repo"
+
     try:
         subprocess.check_call(
-            ["sudo", "qubesctl", "--skip-dom0", "--targets", vm, "state.sls", "fpf-apt-repo"]
+            ["sudo", "qubesctl", "--skip-dom0", "--targets", vm, "state.sls", salt_state]
```

Comment:
looks like this used to be hard-coded to `update.qubes-vm` (https://github.com/freedomofpress/securedrop-workstation/pull/722/files) so just trying to figure out why we now need both `fpf-apt-repo` and `update.qubes-vm`...

Reply:
I think the intent of #722 was to make sure that we can run Debian-specific tasks (such as the `oldstable` tweaks as well as the newly added `autoremove`) whenever we update Debian-based templates. Those tasks aren't run by Qubes' built-in update logic.

Reply:
Oh yeah! Thanks for the explanation -- makes sense now :)